metadata/reportssql/testing_report_functions.sql

```diff
@@ -1269,6 +1269,15 @@ BEGIN
     DECLARE result TINYINT(1) DEFAULT 0;
     DECLARE uuidHIVTestFinalResult VARCHAR(38) DEFAULT "41e48d08-2235-47d5-af12-87a009057603";
 
+    DECLARE uuidHIVTestDate VARCHAR(38) DEFAULT "c6c08cdc-18dc-4f42-809c-959621bc9a6c";
+    DECLARE uuidHIVTestSection VARCHAR(38) DEFAULT "b70dfca0-db21-4533-8c08-4626ff0de265";
+    DECLARE withinReportingPeriod TINYINT(1) DEFAULT 0;
+
+    SET withinReportingPeriod = getObsDatetimeValueInSection(p_patientId, uuidHIVTestDate, uuidHIVTestSection) BETWEEN p_startDate AND p_endDate;
+    IF NOT withinReportingPeriod THEN
```

Comment:
@lauravignoli a quick fix could be to use `IF withinReportingPeriod IS NULL THEN` here